locales/base/translation.json

```diff
@@ -2657,7 +2657,11 @@
       "ageOfPool": "Age of Pool",
       "learnMoreOnProvider": "View Pool on {{providerName}}",
       "deposit": "Deposit",
-      "withdraw": "Withdraw"
+      "withdraw": "Withdraw",
+      "totalDepositAndEarnings": "Total Deposit & Earnings",
+      "earnings": "Earnings",
```

Comment:
Is this `earnings` key being used? I couldn't find any changes here that use it.

Reply:
Removed in 6e8378565de6bbde237606194c87e703ed26abcd!